sigstore/verify/models.py

```diff
@@ -180,7 +180,13 @@ def _verify_bundle(self) -> None:
             raise InvalidBundle(f"unsupported bundle format: {self._inner.media_type}")
 
         # Extract the signing certificate.
-        if media_type == BundleType.BUNDLE_0_3:
+        if media_type in (BundleType.BUNDLE_0_3, BundleType.BUNDLE_0_3_ALT):
+            if media_type == BundleType.BUNDLE_0_3_ALT:
+                _logger.warning(
+                    f"media type {BundleType.BUNDLE_0_3_ALT} is discouraged; "
+                    f"consider using {BundleType.BUNDLE_0_3} for v3 bundles"
+                )
```

Comment:
I understand the intent but this seems unproductive as verifying "historical" bundles is an expected use case. Bundles with BUNDLE_0_3_ALT were correct at the time they were made (as far as this code knows) so should be considered correct now: we should just verify it without complaints and should not imply that a mistake has been made.

Reply:
Fair argument. I'm okay with not warning here; will remove.